apps/src/componentLibrary/common/styles/colors.css

```
@@ -1,17 +1,22 @@
 @import "primitiveColors.css";
 
-/* This file consists of Semantic and Utility colors,
- if you need DSCO colors, you need to import and use this file.
+/* This file consists of Semantic colors,
+ if you need DSCO colors with support of multiple themes, you need to import and use this file.
  */
 
-/* Colors are taken from DISCO Variables New Figma File - Semantic colors and Utility colors variables collections
+/* Colors are taken from DISCO Variables New Figma File - Semantic colors colors variables collection
  via variables2css plugin that generates css code based on Figma variables values and names.
  We're using css here in order to support not only scss, but also css files.
  Also, stylelint fix might be required for autogenerated colors, which can be done in one command.
  */
 
 /* Semantic Colors */
 
+/* Semantic colors values are taken from Primitive Colors (primitiveColors.css).
+ Semantic color value is specified for every theme, meaning that unlike (primitiveColors.css).
+ Semantic colors are meant to be different for every theme.
```

Should this say `…unlike Utility colors (primitiveColors.css)`? Also made a tiny grammar fix.
```suggestion
 Semantic color value is specified for every theme, meaning that unlike Utility colors (primitiveColors.css), Semantic colors are meant to be different for every theme.
```

---

> …unlike Utility colors (primitiveColors.css)

Utility colors are now deleted and deprecated. Primitive Colors are Primitive colors

---

Got it, thanks for clarifying!

---

Updated the original suggestion: 
```suggestion
 Semantic color value is specified for every theme, meaning that unlike Primitive Colors (primitiveColors.css), Semantic colors are meant to be different for every theme.
```

---

Thank you very much! 🤩